### PR TITLE
feat(minifier): Property name generation.

### PR DESCRIPTION
**Pull request by dariajung**
Generate property aliases based on an alphabet of [$, _, 0 - 9, a - z, A - Z]

### CODE REVIEW

**Comment by rkirov** [comment]
LGTM, as reviewed previously, merge it.

File: src/main.ts [hunk 1]
```diff
@@ -77,4 +77,114 @@ export class Minifier {
...
+      'native',
+      'throws',
+      'boolean',
+      'abstract',
+      'volatile',
+      'transient',
+      'synchronized'
+    ];
+
+    return (reserved.indexOf(str) > -1);
```
Comment (mprobst) [inline src/main.ts]: Shouldn't this be a map or object, for O(1) lookups?

File: src/main.ts [hunk 2]
```diff
@@ -77,4 +77,114 @@ export class Minifier {
       }
     }
   }
-}
\ No newline at end of file
+
+  // Alphabet ['$', '_','0' - '9', 'a' - 'z', 'A' - 'Z']
```
Comment (mprobst) [inline src/main.ts]: What does this do if it wraps? Document?

File: src/main.ts (hunk 1 above)
Comment (dariajung) [inline src/main.ts]: Changed to an object literal.

**Pull request merged**